src/MCPServer/lib/assets/workflow.json

```diff
@@ -0,0 +1,14741 @@
+            "group": {
+                "en": "Quarantine",
+                "es": "Cuarentena",
+                "fr": "Quarantaine",
+                "ja": "検疫",
+                "pt_BR": "Quarentena",
+                "sv": "Karantän"
+            }
+        },
+        "087d27be-c719-47d8-9bbb-9a7d8b609c44": {
+            "config": {
+                "@manager": "linkTaskManagerReplacementDicFromChoice",
+                "@model": "MicroServiceChoiceReplacementDic",
+                "replacements": [
+                    {
+                        "description": {
+                            "en": "Identify using Siegfried",
+                            "es": "Identificar usando Siegfried",
+                            "fr": "Identifier en utilisant Siegfried",
+                            "ja": "Identify using Siegfried",
+                            "pt_BR": "Identifique usando Siegfried",
+                            "sv": "Identifiera med hjälp av Sigfried"
+                        },
+                        "id": "25a91595-37f0-4373-a89a-56a757353fb8",
+                        "items": {
+                            "IDCommand": "75290b14-2931-455f-bdde-3b4b3f8b7f15"
+                        }
+                    },
+                    {
+                        "description": {
+                            "en": "Identify using Fido",
+                            "es": "Identificar usando FIDO",
+                            "fr": "Identifier en utilisant Fido",
+                            "ja": "Identify using Fido",
+                            "pt_BR": "Identifique usando Fido",
+                            "sv": "Identifiera med hjälp av Fido"
+                        },
+                        "id": "6f9bfd67-f598-400a-aa2e-12b2657962fc",
+                        "items": {
+                            "IDCommand": "213d1589-c255-474f-81ac-f0a618181e40"
+                        }
+                    },
+                    {
+                        "description": {
+                            "en": "Identify by File Extension",
+                            "es": "Identify by File Extension",
+                            "fr": "Identifier par extensions de fichier",
+                            "ja": "Identify by File Extension",
+                            "pt_BR": "Identificar por extensão de arquivo",
+                            "sv": "Identifiera efter filändelse"
+                        },
+                        "id": "724b17a2-668b-4ef6-9f3b-860d8dfcbb29",
+                        "items": {
+                            "IDCommand": "41efbe1b-3fc7-4b24-9290-d0fb5d0ea9e9"
+                        }
+                    },
+                    {
+                        "description": {
+                            "en": "Skip File Identification",
+                            "es": "Skip File Identification",
+                            "fr": "Passer l'identification du fichier",
+  
```

Comment:
I feel like there's a bit more to be done around the watched directories. At the moment, I can see responsibility for creating them taken by MCP Server, e.g. https://github.com/artefactual/archivematica/blob/a009e15030a08ad7939c986b05587febd295a7ed/src/MCPServer/lib/archivematicaMCP.py#L248-L249, and I can see static references to them, e.g. https://github.com/artefactual/archivematica/blob/a009e15030a08ad7939c986b05587febd295a7ed/src/MCPServer/lib/package.py#L57-L61

They're also created via the `json` workflow via, https://github.com/artefactual/archivematica/blob/a009e15030a08ad7939c986b05587febd295a7ed/src/MCPServer/lib/archivematicaMCP.py#L186-L206

Perhaps there's a way to bring the `package.py` code into the watched directory structure here, e.g. an object that can describe what to do with and without approval `chain_id_auto'? And then remove those other references in the MCP Server? 

The way this manifests itself as a bug is if I create a custom watched directory name in `worfklow.json` e.g. a new name for the standard transfer directory. If I do that, then AM doesn't seem to want to start a transfer, and I haven't quite debugged that part yet. 

I wonder if this change impacts the API too... if I use a custom name in `workflow.json` will `start_transfer` or `unapproved_transfer` need awareness of either of these directories? 

I haven't had a look in that complete level of detail, but if so, an alternative might be to have the watched directory names restricted using the JSON schema instead. So the JSON file can only represent the current set of watched directories.